src/api/java/cvc/CVCApiException.java

```diff
@@ -0,0 +1,9 @@
+package cvc;
```

Comment:
All the files should have the usual boilerplate comments.

Reply:
Changing package name to cvc5, would affect the manually written [files](https://github.com/CVC4/CVC4/pull/6196/files#diff-31a7cb3b95442b3150f94f788c11f51614a5a4577d97e29f003b4ca2fc481af4R13). 
JNI function names include the package name, which would need to be updated for future cvc releases.

Reply:
I actually meant [this](https://github.com/CVC4/CVC4/blob/master/src/api/cpp/cvc5.cpp#L1-L32) part of our files :)

Reply:
Done. Once these java files are stable and handle all needed functionalities, they could also be auto generated.